packages/next/server/render.tsx

```diff
@@ -412,12 +404,6 @@ export async function renderToHTML(
   // next internal queries should be stripped out
   stripInternalQueries(query)
 
-  if (isServerComponent) {
-    throw new Error(
-      'Server Components are not supported from the pages/ directory.'
```

Comment:
I think this is still helpful though, because there're a couple of versions that support RSC in pages.

Reply:
Yeah true 👍